Software/main.py

```diff
@@ -1,257 +1,203 @@
-from tkinter import *
-
-import place as placepackage
-from paho.mqtt import client as mqtt_client
-import random
 import json
-from enum import Enum
-from datetime import datetime
+import random
 import statistics
-import sqlite3
+from datetime import datetime
+
+from paho.mqtt import client as mqtt_client
+
+from Software.model.model import Model
+from Software.model.place import PlaceState, Place
```

Comment:
These two should probably be `from model.[...]` (without `Software.`)

Reply:
Ill try to stick to your suggested convention - fixed.